Tasks/Common/azure-arm-rest/Tests/package-lock.json

```diff
@@ -16,9 +16,9 @@
       "integrity": "sha1-TQJjewZ/6Vi9v906QOxW/vc3Mkc=",
       "dev": true,
       "requires": {
-        "assertion-error": "1.1.0",
-        "deep-eql": "0.1.3",
-        "type-detect": "1.0.0"
+        "assertion-error": "^1.0.1",
+        "deep-eql": "^0.1.3",
```

Comment:
Don't change the version to compatible version format (^). We use exact version to ensure all the dependent modules are also properly IP Scanned.

Reply:
Reverted all the changes in package-lock.json files.